content/docs/command-reference/config.md

```diff
@@ -247,6 +247,77 @@ experiments or projects use a similar structure.
 
 - `exp.live` - path to your [DVCLive](/doc/dvclive) output logs.
 
+### parsing
+
+- `parsing.bool` - Controls the templating syntax for boolean values when used
+  in
+  [dict unpacking](/doc/user-guide/project-structure/pipelines-files#dict-unpacking).
+
+  Valid values are `"store_true"` (default) and `"boolean_optional"`, named
+  after
+  [Python argparse actions](https://docs.python.org/3/library/argparse.html#action).
+
+  Given the following `params.yaml`:
+
+  ```yaml
+  dict:
+    bool-true: true
```

Comment:
Everything else should probably be a separate Example, not part of the `parsing.bool` bullet.